dcos_installer/backend.py

```diff
@@ -26,10 +27,34 @@ def print_messages(messages):
         log.error('{}: {}'.format(key, error))
 
 
+@contextmanager
+def muted_logging():
+    """
+    A helper function that temporarily disables all the logging of the default
+    logger.
+
+    Example use:
+
+        with muted_logging():
+            log.info("Message")
+
+    """
+    logger = logging.getLogger()
+    logger.disabled = True
+
+    yield
+
+    logger.disabled = False
```

Comment:
What if the logger was disabled before entering this context manager? Does this then enable it? If so, that seems like unexpected behaviour.